coalib/coala_delete_orig.py

```diff
@@ -9,18 +9,23 @@
 
 
 def main(log_printer=None, section: Section=None):
+    configure_logging()
+
+    return delete_orig(log_printer, section)
+
+
+def delete_orig(log_printer=None, section: Section=None):
     """
     >>> import logging
     >>> from coalib.output.Logging import configure_logging
     >>> configure_logging(stdout=True, color=False)
     20
-    >>> main(log_printer=SomeLogPrinter)
+    >>> delete_orig(log_printer="")
     [WARNING][... Using log_printer is deprecated...
     """
     if log_printer is not None:
         logging.warning('Using log_printer is deprecated, '
-                        'Use python inbuilt logging instead.')
-    configure_logging()
+                        'Using python inbuilt logging instead.')
```

Comment:
Oh! I actually added that warning to `main` and then changed moved it to `delete_orig` in the commit I added that. 
But the 2nd time I forgot to change it to `use`